src/core/index.test.js

```diff
@@ -1,6 +1,8 @@
-const { core } = require(`../../test/index.js`);
+import chai from 'chai';
+import core from './index.js';
 
```

Comment:
Should we add Collection tests here since Collection is within `src/core/`

Reply:
Yup - totally forgot.